core/src/main/java/io/questdb/cutlass/text/ParallelCsvFileImporter.java

```diff
@@ -962,7 +985,8 @@ void phaseIndexing() throws TextException {
 
     private void phasePartitionImport() throws TextImportException {
         if (partitions.size() == 0) {
-            throw TextImportException.instance(TextImportTask.PHASE_PARTITION_IMPORT, "No partitions to merge and load found");
+            throw TextImportException.instance(TextImportTask.PHASE_PARTITION_IMPORT,
+                    "No partitions to merge and load found. Possible reasons: timestamp format mismatch or no rows in input file.");
```

Comment:
could we check the "no rows in the input file" eagerly? before this phase.

Reply:
Yeah, we could do this. Let me add that.

Reply:
Done in dfcfd6f63e7dc23543effe4c00c240af08a31116

As a nice side effect, from now on we don't swallow `CsvFileIndexer`'s errors. They're reported in phase 2's log row.